src/shared/components/Views/ViewStatisticsProgress.tsx

```diff
@@ -108,8 +108,7 @@ export const ViewStatisticsContainer: React.FunctionComponent<ViewStatisticsCont
     ).subscribe(
       // @ts-ignore
       // TODO fix for v1.3 release
```

Comment:
is this comment obsolete? ;)

Reply:
+1

Reply:
No, it will be after we update the nexus-sdk package to v1.3.1 with the statistics changes (so we dont have to use ts-ignore)